src/operator/subgraph/mkldnn/mkldnn_fc_property.h

```diff
@@ -120,19 +120,19 @@ class SgMKLDNNFCSelector : public SubgraphSelector {
 class SgMKLDNNFCProperty : public SubgraphProperty {
  public:
   SgMKLDNNFCProperty() {
-    disable_all = dmlc::GetEnv("MXNET_DISABLE_MKLDNN_OPT", false);
     disable_fc_relu = dmlc::GetEnv("MXNET_DISABLE_MKLDNN_FUSE_FC_RELU", false);
-
-    disable_all = disable_all || disable_fc_relu;
-    if (disable_all) {
-      LOG(INFO) << "MKLDNN FullyConnected optimization pass is disabled.";
-    } else {
-      LOG(INFO) << "Start to execute MKLDNN FullyConnected optimization pass.";
-    }
   }
 
   static SubgraphPropertyPtr Create() {
-    return std::make_shared<SgMKLDNNFCProperty>();
+    static const std::string &name = "MKLDNN FullyConnected optimization pass";
+    if (dmlc::GetEnv("MXNET_DISABLE_MKLDNN_FC_OPT", 0)) {
```

Comment:
can you document all these variables been documented in env var docs.

Reply:
Added